src/test/objectstore/store_test.cc

```diff
@@ -3560,6 +3560,16 @@ class SyntheticWorkloadState {
     return status;
   }
 
+  void fsck() {
+    Mutex::Locker locker(lock);
+    EnterExit ee("fsck");
+    while (in_flight)
+      cond.Wait(lock);
+    store->umount();
+    store->fsck();
```

Comment:
We have fsck_on_(u)mount set to true for this test suite (see main func) hence there is no need to call fsck directly. That's just a waste of time...